spec/models/rent_spec.rb

```diff
@@ -2,239 +2,171 @@
 require 'rails_helper'
 
 RSpec.describe Rent, type: :model do
-  subject(:rent) { build(:rent, user: create(:user, member_at: nil)) }
-  let(:saved) { create(:rent, :good) }
+  let(:member) { create(:user) }
+  let(:n_member) { create(:user, :not_member) }
+
+  let(:council) { create(:council) }
+
+  let(:new_rent) { build(:rent, user: member) }
+  let(:new_rent_council) { build(:rent, user: member, council: council) }
+  subject(:rent_n) { build(:rent, user: n_member) }
+
+  let(:rent) { create(:rent, user: member, d_from: Time.zone.now + 15.days,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.